business_requirement_deliverable_resource_template_categ/data/business_requirement_deliverable_resource_template_categ_demo.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data noupdate="1">
+
+        <record id="business_requirement_deliverable_resource_template.resource_template_06"
+                model="business.requirement.resource.template">
+                <field name="categ_id"
```

Comment:
Could you please fix the indentation of this line? `field` is child of `record` (same for all the document)